### PR TITLE
Added MeetMe project

### PR DESCRIPTION
**Pull request by anasb**
MeetMe project (team: Mark A., Anas B., Zaid H.).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ See [Project Logistics](projects/logistics.md) for important dates and deliverab
 * [Project Deets](projects/project-deets.md) - **Team:** [Michael Wang](people/michael-wang.md), [Vivek Sudarsan](people/vivek-sudarsan.md), [Julian Ferdman](people/julian-ferdman.md)
 * [Social Timeline](projects/social-timeline.md) - **Team:** [Inna Shteinbuk](people/inna-shteinbuk.md), [Jocelyn Kong](people/jocelyn-kong.md)
 * [Wanderluster](projects/wanderluster.md) - **Team:** [Shimiao Huang](people/shimiao-huang.md), [Yanbo Li](people/yanbo-li.md), [Liudan Xiao](people/liudan-xiao.md)
+* [MeetMe](projects/meetme.md) - **Team** [Mark Assad](people/mark-assad.md), [Anas Bouzoubaa](people/anas-bouzoubaa.md), Zaid Haque
```
Comment (dblock) [inline README.md]: Zaid doesn't have a page? He needs to.

File: projects/meetme.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+MeetMe
+--------------------
+
+### Background:
+
+Automating the process of scheduling and keeping track of meetings has been around for a long time, especially in professional environments. Typically, organizations make use of products such as Outlook or Google Calendar to achieve this. However, these either require all users to be on a certain app/network, or require users to share contacts/details with each other, which is troublesome, especially for one-off meetings. They are also not tailored for individuals.
+MeetMe is an app that syncs with all your calendars to seamlessly schedule meetings, whether a professional one or for drinks with friends.
```
Comment (dblock) [inline projects/meetme.md:8]: I am pretty sure that in Markdown this needs to have another empty line if you want it on a separate line.

**Comment by anasb** [comment]
@dblock Fixed the line issue, thanks. Zaid will add his page asap.

File: README.md (hunk 1 above)
Comment (anasb) [inline README.md]: It's in #154

Comment (dblock) [inline README.md]: I merged it. Update this PR to link his name.

**Comment by anasb** [comment]
@dblock Updated the PR and squashed all commits.

**Pull request merged**